completion/available/fabric.completion.bash

```diff
@@ -62,9 +64,11 @@ function __fab_chache_mtime() {
 function __fab_fabfile_mtime() {
 	local f="fabfile"
 	if [[ -e "$f.py" ]]; then
+		# shellcheck disable=SC2090
 		${__FAB_COMPLETION_MTIME_COMMAND} "$f.py" | xargs -n 1 expr
 	else
 		# Suppose that it's a fabfile dir
+		# shellcheck disable=SC2086,SC2090,SC2038
 		find $f/*.py -exec ${__FAB_COMPLETION_MTIME_COMMAND} {} + \
```

Comment:
```suggestion
		# shellcheck disable=SC2086,SC2090,SC2038
		find "$f"/*.py -exec ${__FAB_COMPLETION_MTIME_COMMAND} {} + \
```

Reply:
that one I was pretty annoyed with. the `*.py` I was not happy about, it should make sense only in a `-name` param but the first param on a find is a name of a directory, not a list of filenames, so that looks like a coding mistake. I thought it might be safe to leave it like that first till I figure out what the original code was supposed to actually do.

Reply:
ok, I see. `find $dir/*.ext` is really the same as `find $dir -name "*.ext" -depth 1`
did I get that right? I had no ides that syntax is supported.

Reply:
> ok, I see. `find $dir/*.ext` is really the same as `find $dir -name "*.ext" -depth 1` did I get that right?

Yes, they are equivalent, though I'm not sure if `-depth 1` is intentional as you suspected in the first comment. The original intent might have been actually "$f" -name '*.py' to list `.py` files in all the levels.

Anyway, there is no reason to leave `$f` unquoted here.